scripts/generate-clients/copy-to-clients.js

```diff
@@ -25,6 +32,28 @@ const getOverwritablePredicate = packageName => pathName => {
   );
 };
 
+const mergeManifest = (fromContent, toContent) => {
+  const merged = {};
+  const toNames = Object.keys(toContent);
+  for (const name of toNames) {
+    if (!fromContent[name]) {
+      merged[name] = toContent[name];
```

Comment:
This line is not required, as if the key (say dependency) is removed from generated code it should be removed in package.json
If we're manually adding any keys in clients package.json, it should be updated in codegen